contrib/hyperscan/matching/input_matchers/source/BUILD

```diff
@@ -30,7 +30,7 @@ envoy_cmake(
         "CMAKE_BUILD_TYPE": "Release",
         "CMAKE_INSTALL_LIBDIR": "lib",
         "FAT_RUNTIME": "on",
-        "RAGEL": "$EXT_BUILD_DEPS/ragel/bin/ragel",
+        "RAGEL": "$$EXT_BUILD_DEPS$$/ragel/bin/ragel",
```

Comment:
also not sure why this needs this incantation - but from deciphering the error and looking at upstream it seems to be correct, and works